### ISSUE TITLE
Inconsistency in ALCARECOHcal paths

### ISSUE THREAD

**Issue by Dr15Jones** [cms-sw/cmssw#34777]
The new schedule consistency checker, #34735 ,  found the following problem in RelVal 1000.0
```
----- Begin Fatal Exception 03-Aug-2021 23:03:32 CEST-----------------------
An exception of category 'ScheduleExecutionFailure' occurred while
   [0] Calling beginJob
Exception Message:
Unrunnable schedule
Paths are non consistent
  module 'ALCARECOHcalCalPhisymDQM' depends on 'hbherecoNoise' which appears on paths
  pathALCARECOHcalCalMinBias 
but is missing from
  pathALCARECOHcalCalIterativePhiSym 
----- End Fatal Exception -------------------------------------------------
```

looking at the configuration, `hbherecoNoise` only appears on `pathALCARECOHcalCalMinBias` which means `pathALCARECOHcalCalIterativePhiSym` can not operate without that other path being present.

**Comment by Dr15Jones** [comment]
assign alca

**Comment by Dr15Jones** [comment]
The configuration fragments are

https://github.com/cms-sw/cmssw/blob/6d2f66057131baacc2fcbdd203588c41c885b42c/Calibration/HcalAlCaRecoProducers/python/ALCARECOHcalCalMinBias_cff.py#L68

https://github.com/cms-sw/cmssw/blob/6d2f66057131baacc2fcbdd203588c41c885b42c/Configuration/StandardSequences/python/AlCaRecoStream_Specials_cff.py#L47

https://github.com/cms-sw/cmssw/blob/cfbe41337d7eb1acb1aa032038770bb5862751f3/Configuration/StandardSequences/python/AlCaRecoStreams_cff.py#L218

**Comment by tvami** [comment]
Hi @bsunanda can you please look into this? Thanks!

**Comment by Dr15Jones** [comment]
One thing to know, the Paths `pathALCARECOHcalCalMinBias` and `pathALCARECOHcalCalIterativePhiSym` both start with different filters. If the filter on Path `pathALCARECOHcalCalMinBias` fails and the filter on path `pathALCARECOHcalCalIterativePhiSym` succeeds, then when `ALCARECOHcalCalPhisymDQM` is called, the EDAnalyzer will fail to get the data products it needs (since they were only defined on Path `pathALCARECOHcalCalMinBias`).

**Comment by bsunanda** [comment]
The two paths produce different outputs and the same DQMAnalyzer should not be used for both. I am trying to see if a second DQM analyzer can be made for pathALCARECOHcalCalIterativePhiSym

**Comment by bsunanda** [comment]
Submitted a PR #34784 to resolve this issue.

**Comment by perrotta** [comment]
As this isssue is fixed by #34784, it can probably be signed and closed

**Comment by francescobrivio** [comment]
+alca
